tests/test_publish.py

```diff
@@ -1289,3 +1290,50 @@ def test_update_database_without_media(tmpdir, persistent_repository):
             audeer.path(build_root, attachment_file),
             audeer.path(db_load.root, attachment_file),
         )
+
+
+def test_unstability_of_media_files_order_during_publication():
+    media_files = {}
+    n_iterations = 10
+    for i in range(n_iterations):
```

Comment:
The bug comes from the fact that when publication is called different times, the order change so sometimes they are sorted, and other times they're not, so it seemed like repeating the process multiple times is the way to test the stability, or do you suggest a different/better approach?

Reply:
It's a test of time :) If it not always sorted, someday it will fail.

Reply:
Alright then maybe iterating multiple times was an offline solution to check if it's working correctly, and since we know it's working we can just test that it's sorted. I've changed it accordingly.